crm_solver/atomic_db.py

```diff
@@ -5,6 +5,7 @@
 from scipy.interpolate import interp1d
 import matplotlib.pyplot as plt
 import pandas
+import os
```

Comment:
To be checked if needed

Reply:
It is redundant. Was removed in d4967ea.